substrate-archive/src/database/models.rs

```diff
@@ -134,3 +136,114 @@ impl<Hash: Copy> From<BatchStorage<Hash>> for Vec<StorageModel<Hash>> {
 		original.inner.into_iter().flat_map(Vec::<StorageModel<Hash>>::from).collect()
 	}
 }
+
+/// Config that is stored/restored in
+/// Postgres on every run.
+#[derive(FromRow, Debug, Clone)]
+pub struct PersistentConfig {
+	// internal SQL identifier
+	id: i32,
+	/// RabbitMQ Queue Name
+	pub task_queue: String,
+	/// Last time the archive was run
+	pub last_run: DateTime<Utc>,
+	pub major: i32,
+	pub minor: i32,
+	pub patch: i32,
```

Comment:
Yeah. This Config is mostly here to store the name/options of the queue in RabbitMq, so that we can persist the queue across archive runs/reboots, and save ourselves from querying Postgres even more. 

I documented these versions here: https://github.com/paritytech/substrate-archive/pull/327/commits/ee1366f47c4b4a4c7adaf20c077b48484a8efd4e

They are the archive library version. They aren't necessary for the task queue, but they are useful for debugging and inserting a few extra bytes into the database once every run isn't a big deal afaict.